File: P32151/Chernykh Roman/Lab6/function.py
import math


def get_func(num):
    if num == 1:
        return lambda x, y: math.cos(y) + 6 * x
    if num == 2:
        return lambda x, y: math.sin(x ** 2) + y + 6
    if num == 3:
        return lambda x, y: x**2 - 6 * y**2
    if num == 4:
        return lambda x, y: (x - y) ** 3


def get_str_func(num):
    if num == 1:
        return "cos(y) + 6x"
    if num == 2:
        return "sin(x^2) + y + 6"
    if num == 3:
        return "x^2 - 6y^2"
    if num == 4:
        return "(x - y)^3"

File: P32082/Rogachev/lab5/Comp5/src/main/java/Function.java
import static java.lang.Math.sin;
public class Function {
    public static double getFunction(double x, int num) {
        switch (num) {
            case 0 -> {
                return sin(x);
            }
            case 1 -> {
                return x * x + 5 * x + 3;
            }
            case 2 -> {
                return x;
            }
        }
        return 0;
    }
}
